packages/ruleset/package.json

```diff
@@ -45,8 +45,8 @@
     "coverageDirectory": "./coverage/",
     "testEnvironment": "node",
     "moduleNameMapper": {
-      "nimma/legacy": "<rootDir>/../../node_modules/nimma/dist/legacy/cjs",
-      "nimma/fallbacks": "<rootDir>/../../node_modules/nimma/dist/cjs/fallbacks/"
+      "nimma/legacy": "<rootDir>/../../node_modules/@stoplight/spectral-core/node_modules/nimma/dist/legacy/cjs",
```

Comment:
After adding the override and running `npm install`, a bunch of tests failed due to the fact that nimma was no longer available at the same location.  It's now located within the spectral-core's own node_modules directory for some reason.  Modifying these mapper entries seemed to let all the tests run clean.